docs/control-structures.rst

```diff
@@ -614,6 +614,16 @@ or return values from calls to external contracts.
 A ``Error(string)`` exception is generated in the following situations:
 
 #. Calling ``require`` with an argument that evaluates to ``false``.
+#. If you perform an external function call targeting a contract that contains no code.
```

Comment:
Is this true? The yul code seems to only do `if iszero(extcodesize(expr_20_address)) { revert(0, 0) }`

Reply:
Most of these do not provide a reason string. Yes, the description here is a little inaccurate, but I think in the beginning, it mentions that empty data also counts as an "Error"-revert.

Reply:
I'll clarify it.